web/modules/content_moderation_notifications/src/Form/ContentModerationNotificationsEditForm.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
I'm not sure about all these PHP files of the modules. Are they really supposed to be committed to git? I think it bloats the PRs a lot?

Reply:
The Drupal Module folder which are located away from the core are either contributor modules or custom modules. We need them in the git repo. But as expressed they will lessen drastically over time.